### PR TITLE
Added Class for Emails from Server

### PR DESCRIPTION
**Pull request by shivenmian**
This is for issue #361. Created the class. Will develop on the server functionality later. Please check @Orbiter @treba123 @mariobehling

### CODE REVIEW

File: src/org/loklak/LoklakEmailHandler.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+import org.eclipse.jetty.util.log.Log;
+
+public class LoklakEmailHandler {
+	
+	private String addressTo;
+	private String addressFrom;
+	private String subject;
```
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Avoid unused private fields such as 'subject'.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l31)

File: src/org/loklak/LoklakEmailHandler.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	private Pattern pattern;
+	private static final String EMAIL_PATTERN = "^[_A-Za-z0-9-\\+]+(\\.[_A-Za-z0-9-]+)*@" + "[A-Za-z0-9-]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$";
+	
+	public LoklakEmailHandler (String addressTo, String addressFrom, String subject, String text) throws PatternSyntaxException, IllegalStateException{
+		pattern = Pattern.compile(EMAIL_PATTERN);
+		this.addressTo = addressTo;
+		this.addressFrom = addressFrom;
+		this.subject = subject;
+		this.text = text;
+		if(pattern.matcher(addressTo).matches() && pattern.matcher(addressFrom).matches()){
```
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Avoid empty if statements](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l42)

File: src/org/loklak/LoklakEmailHandler.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import java.util.regex.PatternSyntaxException;
+
+import org.eclipse.jetty.util.log.Log;
+
+public class LoklakEmailHandler {
+	
+	private String addressTo;
+	private String addressFrom;
+	private String subject;
+	private String text;
```
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Avoid unused private fields such as 'text'.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l32)

File: src/org/loklak/LoklakEmailHandler.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+import org.eclipse.jetty.util.log.Log;
+
+public class LoklakEmailHandler {
+	
+	private String addressTo;
+	private String addressFrom;
```
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Perhaps 'addressFrom' could be replaced by a local variable.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l30)

File: src/org/loklak/LoklakEmailHandler.java (hunk 3 above)
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Perhaps 'text' could be replaced by a local variable.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l32)

File: src/org/loklak/LoklakEmailHandler.java (hunk 4 above)
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Avoid unused private fields such as 'addressFrom'.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l30)

File: src/org/loklak/LoklakEmailHandler.java (hunk 1 above)
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Perhaps 'subject' could be replaced by a local variable.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l31)

File: src/org/loklak/LoklakEmailHandler.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+	private String subject;
+	private String text;
+	private Pattern pattern;
```
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Perhaps 'pattern' could be replaced by a local variable.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l33)

File: src/org/loklak/LoklakEmailHandler.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+package org.loklak;
+
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+import org.eclipse.jetty.util.log.Log;
+
+public class LoklakEmailHandler {
+	
+	private String addressTo;
```
Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Perhaps 'addressTo' could be replaced by a local variable.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l29)

Comment (sudheesh001) [inline src/org/loklak/LoklakEmailHandler.java]: Codacy Issue found: [Avoid unused private fields such as 'addressTo'.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2629824425/issues/source?bid=3288751&fileBranchId=3317765#l29)

**Comment by Orbiter** [comment]
@sudheesh001 please turn this automatic notes off, they are misleading and irrelevant

**Comment by shivenmian** [comment]
@Orbiter anything to be added to this?

**Comment by shivenmian** [comment]
@Orbiter anything to be added to this (apart from email server) or is it fine?

**Comment by shivenmian** [comment]
@Orbiter please check.

**Comment by shivenmian** [comment]
@Orbiter please check if it's ok.

**Comment by Orbiter** [comment]
Well thats just a stub class! Please extend this so it will be possible to actually send messages, this class just has an initializer.

**Comment by shivenmian** [comment]
@Orbiter Yes, in this code I added in comments ```//send to server```. Will work on that next. Just wanted to ensure this is ok.

**Pull request merged**